daprdocs/content/en/reference/components-reference/supported-state-stores/setup-inmemory.md

```diff
@@ -0,0 +1,29 @@
+---
+type: docs
+title: "In Memory"
+linkTitle: "In Memory"
+description: "Detailed documentation on the In Memory state component"
+aliases:
+  - "/operations/components/setup-state-store/supported-state-stores/setup-inmemory/"
+---
+
+The In Memory state store component is useful for development purposes and works inside of a single machine boundary.
```

Comment:
Q: Can this ever be used for production? In other words, can this component get to stable state and be used in production on a single machine, or is that never a goal?

Reply:
Seems like a bad idea because it would increase the footprint of the Daprd process indefinitely. In a container world this would certainly mean the size of the sidecar container needs to be increased and that isn't useful.

Reply:
What does Stable mean here? Inherently it's a development feature only right?

I suppose some people might want the in memory state store to be used in production on some IOT devices? that could make sense...

Reply:
Then should there be a way of marking components to say developer only and will never be stable. Stable is simple N/A?

Reply:
>Q: Can this ever be used for production? 

For now, it's not a goal. I will follow your advice and mark it as N/A.

But, when developing other sidecar project, I have encountered a scenario in the production environment that users need "local cache" in the sidecar. I think this component can be improved to be a local cache.
Anyway, it depends on user request. We can wait for users' feedback